src/OrganisationRegistry.UI/app/organisation/detail/detail-routing.module.ts

```diff
@@ -676,6 +681,24 @@ const routes: Routes = [
           }
         ]
       },
+      {
+        path: 'manage', component: OrganisationOpeningHoursComponent,
```

Comment:
`OrganisationOpeningHoursComponent` is this correct? Doesn't seem to be about opening hours at all